code/network/multi_sexp.cpp

```diff
@@ -9,8 +9,8 @@
 #include "network/multi.h"
 #include "network/multiutil.h"
 
-#define CALLBACK_TERMINATOR	255
-int TEMP_DATA_SIZE = -1;
+static const std::uint8_t CALLBACK_TERMINATOR = 255;
+static const short TEMP_DATA_SIZE = -1;
```

Comment:
If this needs to be 2 bytes across all systems, then use an int16_t. "short" "int" and "long" are still subjective to the architecture.

Reply:
There is a safety check in the ADD_* macros to make sure the size is correct, so that shouldn't be a problem. The code won't even compile if the type size doesn't match what's expected.

Still, I would normally use a properly sized data type for this and I'm not sure why I didn't.